R/globals.R

```diff
@@ -0,0 +1,24 @@
+# this is needed for non-standard evaluation Multiple links suggest using
```

Comment:
Thanks for this -- I will need to read up on this more. It also leaves me wondering though if we should rewrite the package to get rid of the non-standard evaluation. I'm not sure I really had a great design vision in mind when I wrote it this way. Curious if you have any thoughts.